packages/grafana-ui/src/components/Slider/Slider.test.tsx

```diff
@@ -1,4 +1,6 @@
-import { mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { UserEvent } from '@testing-library/user-event/dist/types/setup';
```

Comment:
Strange that it is not exported directly by the package.

Reply:
yeah agreed it's annoying. it's like they didn't consider/don't want to encourage using it in a `beforeEach` 🤔